bundles/mapping/drawtools/request/StopDrawingRequest.js

```diff
@@ -16,11 +16,15 @@ Oskari.clazz
          *        id for drawing as given in StartDrawingRequest
          * @param {boolean} clearCurrent
          *        true to remove the current drawing
+         * @param {boolean} clearCurrent
+         *        true to remove the current drawing
          */
 
-        function(id, clearCurrent) {
+        function(id, clearCurrent, sendEvent) {
```

Comment:
Sending the event is the default thing to do and you don't want to have 99% cases of using this request to have to include true as a third param. sendEvent should be suppressEvent which can be used in the corner cases where you actually don't want an event about the finalized geometry. Also when changing requests/events make sure you update the API-docs regarding the request/event and mention the change in api/CHANGELOG.md